src/data/faq_de.json

```diff
@@ -800,71 +800,11 @@
                                 "anchor": "val_service_basics",
                                 "active": false,
                                 "textblock": [
+                                    "<b>AKTUELL 10. Januar 2022</b><br/>Bereits seit Oktober 2021 liegen die europaweit gültigen Anforderungen an Onlineverifikationsdienste vor. In Deutschland ist momentan <b>noch kein Anbieter eines Onlineverifikationsdienstes zugelassen</b> worden. Derzeit erfolgen Tests, deren Ergebnisse zunächst abgewartet werden. Erst nach einer Auswertung dieser Tests und nach erfolgter Beratung durch den Bundesbeauftragten für den Datenschutz und die Informationsfreiheit (BfDI), werden Onlineverifikationsdienste in der Praxis verfügbar sein. Mit Version 2.15 ist die CWA aber bereit, entsprechende Verifikationsdienste zu unterstützen, sobald Anbieter eines solchen Dienstes zugelassen sind.<hr/>",

```

Comment:
This is only a placeholder. The Open-Source-Team should provide a better text as update, to reflect the current situation.